examples/README.md

```diff
@@ -15,7 +15,8 @@ md_toc github examples/README.md | sed -e "s/\s-\s/ * /"
   * [hpc-enterprise-slurm.yaml](#hpc-enterprise-slurmyaml-) ![core-badge]
   * [hpc-slurm6-tpu.yaml](#hpc-slurm6-tpuyaml--) ![community-badge] ![experimental-badge]
   * [ml-slurm.yaml](#ml-slurmyaml-) ![core-badge]
-  * [image-builder.yaml](#image-builderyaml-) ![core-badge]
+  * [image-builder.yaml](#image-builderyaml--) ![core-badge] ![experimental-badge]
+  * [image-builder-v5-legacy.yaml](#image-builderyaml--) ![core-badge]
```

Comment:
We need to rename as discussed.  
v6: <bp>-v6.yaml
v5: <bp>.yaml

Reply:
Done.